.github/workflows/no_old_refs.yml

```diff
@@ -0,0 +1,55 @@
+#####################################################################################
+# Github Action to check references to closed issues in code.
+#
+# Workflow starts when:
+# 1) push on master branch
+#
+#####################################################################################
+name: "Check no closed issue references"
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  check_issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download checkstyle
+        uses: actions/checkout@v2
+        with:
+          repository: checkstyle/checkstyle
+          ref: master
+          path: checkstyle
+
+      - name: Check Issues
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+          MENTIONED_ISSUES: '/tmp/mentioned_issues'
+          CLOSED_ISSUES: '/tmp/failed_issues'
+          API_GITHUB_PREFIX: 'https://api.github.com/repos'
+          GITHUB_HOST: 'https://github.com'
+        run: |
+          cd checkstyle
+
+          # collect issues where full link is used
```

Comment:
In next PR , please move logic back to she'll file.
All CIs and GitHub actions are also CI should have in their config only their specific logic on preparation of environment or getting data to execute she'll script, this will allt to test script on local

Reply:
I tried to run it from file, but failed. It looks like action does not want to run external files, therefore I inlined all logic into action

Reply:
I found an issue with execution, I will change it back to sh script